cve_bin_tool/checkers/libevent.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for bash
+https://www.cvedetails.com/product/32303/Libevent-Project-Libevent.html?vendor_id=15590
+"""
+from . import Checker
+
+
+class LibeventChecker(Checker):
+    CONTAINS_PATTERNS = [r"Abstract asynchronous event notification library"]
+    FILENAME_PATTERNS = [r"libevent-2.1.so."]
+    VERSION_PATTERNS = [
+        r"([0-9]+\.[0-9]+\.[0-9]+)-stable",
+        r"([0-9]+\.[0-9]+\.[0-9][0-9]+)-stable",

```

Comment:
So, anything this pattern matches will also match the pattern above it.  Let's distill it a bit: The different part is

`[0-9]+`

which means "match one or more numbers consisting of the digits 012345679.  So it'll match 1 or 12 or 9823.  It will match any one digit or more whole positive number.

versus
`[0-9][0-9]+`

This says "match exactly one digit 0-9, then match one or more of the same".  So it won't match 1, but it would match 12 or 9823.  It'll match any two digit or more whole positive number.

So if we already have something that matches any one digit or more positive number, that'll cover any two digit or more positive number, so no need for two regexes here.

Regexes have a reputation for being hard, because they *can* get pretty complicated, but the ones we use for cve-bin-tool are generally fairly straightforwards once you get used to the operators.  If you haven't taken the time yet, you might find the `re` documentation fairly helpful: https://docs.python.org/3/library/re.html

You should also try testing your regexes on the console if you're not super comfortable with them yet.   type "python" with no arguments to get into the python shell (it should give you a `>>>` prompt) and you can mess around re.match with them directly (just don't forget to type `import re` first) to try stuff out.


```python
C:\Users\terrioda>python
Python 3.8.5 (tags/v3.8.5:580fbb0, Jul 20 2020, 15:57:54) [MSC v.1924 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import re
>>> test_pattern1 = r"[0-9]+"
>>> test_pattern2 = r"[0-9][0-9]+"
>>> number_strings = ["0", "12", "123", "1234", "12345"]
>>> for ns in number_strings:
...      print(f"{ns} is matched by...")
...      if re.match(test_pattern1, ns):
...          print("Pattern 1")
...      if re.match(test_pattern2, ns):
...          print("Pattern 2")
...
0 is matched by...
Pattern 1
12 is matched by...
Pattern 1
Pattern 2
123 is matched by...
Pattern 1
Pattern 2
1234 is matched by...
Pattern 1
Pattern 2
12345 is matched by...
Pattern 1
Pattern 2

Reply:
```suggestion

```

That was a long comment, but the end result: you can delete this line, it's not doing anything.